scrc/preprocessors/text_to_database.py

```diff
@@ -89,14 +89,15 @@ def build_dataset(self) -> List[dict]:
         return all_processed_files
     
     def filter_by_text_length(self, spider_dict_list: dict):
-        filtered = [x for x in spider_dict_list if (len(x['pdf_raw']) > 500 or len(x['html_raw']) > 500)]
+        filtered = [x for x in spider_dict_list if (len(x['pdf_raw']) > 1000 or len(x['html_raw']) > 1000)]
```

Comment:
is it possible to replace this magic number with some constant?

Reply:
I replaced it with a constant